WSSiOS/WSSiOS/Source/Presentation/Search/NormalSearch/NormalSearchViewController/NormalSearchViewController.swift

```diff
@@ -102,16 +170,43 @@ final class NormalSearchViewController: UIViewController {
                 owner.rootView.resultView.updateCollectionViewHeight(height: height)
             })
             .disposed(by: disposeBag)
+        
+        output.normalSearchCellEnabled
+            .subscribe(with: self, onNext: { owner, indexPath in
+                //TODO: API 연결 후 수정 예정
+                owner.pushToDetailViewController(novelId: 0)
+            })
+            .disposed(by: disposeBag)
+        
+        rootView.headerView.backButton.rx.tap
+            .throttle(.seconds(3), latest: false, scheduler: MainScheduler.instance)
+            .subscribe(with: self, onNext: { owner, _ in
+                owner.popToLastViewController()
+            })
+            .disposed(by: disposeBag)
+        
+        rootView.resultView.normalSearchCollectionView.rx.swipeGesture(.up)
+            .when(.recognized)
+            .subscribe(with: self, onNext: { owner, _ in
+                self.view.endEditing(true)
+            })
+            .disposed(by: disposeBag)
+        
+        rootView.resultView.normalSearchCollectionView.rx.swipeGesture(.down)
+            .when(.recognized)
+            .subscribe(with: self, onNext: { owner, _ in
+                self.view.endEditing(true)
+            })
+            .disposed(by: disposeBag)
```

Comment:
W2
하나로 합칠 수 이써요
글고 얘네도 뷰모델로 빼면 좋을 거 같아요!
```suggestion
        rootView.resultView.normalSearchCollectionView.rx.swipeGesture([.up, .down])
            .when(.recognized)
            .subscribe(with: self, onNext: { owner, _ in
                self.view.endEditing(true)
            })
            .disposed(by: disposeBag)
```

Reply:
뷰모델로 분리 했습니당! 감사해용